src/main/java/com/cegeka/academy/web/rest/eventController/EventController.java

```diff
@@ -0,0 +1,26 @@
+package com.cegeka.academy.web.rest.eventController;
+
+import com.cegeka.academy.domain.Event;
+import com.cegeka.academy.domain.User;
+import com.cegeka.academy.service.EventService;
+import com.cegeka.academy.service.dto.EventDTO;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("event")
+public class EventController {
+    private final EventService eventService;
+    @Autowired
+    public EventController(EventService eventService) {
+        this.eventService = eventService;
+    }
+
+    @PutMapping("/update")
```

Comment:
Request-uri de genul PUT /_entityName_ sunt suficiente ca sa spuna "asta e un update".
Vezi [aici](https://martinfowler.com/articles/richardsonMaturityModel.html) nivelurile de la 0 la 2, nivelul 3 nu intra in discutie.

Request detaliat pentru un update:
PUT http://localhost:8080/event {_body_}